tools/dind/config/services/resin-supervisor-dind.service

```diff
@@ -8,7 +8,7 @@ Before=openvpn-resin.service
 WorkingDirectory=/usr/src/app
 EnvironmentFile=/usr/src/app/config/env
 EnvironmentFile=/usr/src/app/config/localenv
-ExecStartPre=/bin/bash -c 'if [ "${PRELOADED_IMAGE}" == "true" ]; then /usr/bin/docker pull $(jq ".[0].imageId" ${APPS_PATH}); fi'
+ExecStartPre=/bin/bash -c 'if [ "${PRELOADED_IMAGE}" == "true" ]; then /usr/bin/docker pull $(jq ".[0].imageId" ${APPS_PATH} | sed \'s/"//g\'); fi'
```

Comment:
Can you change this to
```
jq --raw-output ".[0].imageId" ${APPS_PATH}
```
please, which means that jq outputs the string contents by themselves rather than a JSON version (which may have escaped characters in, eg it will print `\n` rather than an actual new line)